### PR TITLE
add rule package_libselinux_installed

### LINKED ISSUE #5223: CIS 1.7.1.1 Ensure SELinux is installed (Scored)
[CIS_Red_Hat_Enterprise_Linux_8_Benchmark_v1.0.0 (1).pdf](https://github.com/ComplianceAsCode/content/files/4301017/CIS_Red_Hat_Enterprise_Linux_8_Benchmark_v1.0.0.1.pdf)

Need rule to see if SELinux is installed

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

Add templated rule. Add the rule to rhel8 cis.

#### Rationale:

cis efford

fixes #5223

### CODE REVIEW

File: linux_os/guide/system/selinux/package_libselinux_installed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: 'Install libselinux Package'
+
+description: |-
+    {{{ describe_package_install(package="libselinux") }}}
+
+rationale: |-
+    Security-enhanced Linux is a feature of the Linux kernel and a number of utilities
+    with enhanced security functionality designed to add mandatory access controls to Linux.
+    The Security-enhanced Linux kernel contains new architectural components originally
+    developed to improve security of the Flask operating system. These architectural components
```
Comment (jan-cerny) [inline linux_os/guide/system/selinux/package_libselinux_installed/rule.yml]: What is Flask?

Comment (vojtapolasek) [reply linux_os/guide/system/selinux/package_libselinux_installed/rule.yml]: https://en.wikipedia.org/wiki/FLASK

**Comment by vojtapolasek** [comment]
fixed conflicts

File: linux_os/guide/system/selinux/package_libselinux_installed/rule.yml (hunk 1 above)
Comment (redhatrises) [inline linux_os/guide/system/selinux/package_libselinux_installed/rule.yml]: Please remove this entire sentence mentioning Flash. It is not needed in the rationale at all and will serve to create unnecessary confusion.

Comment (vojtapolasek) [reply linux_os/guide/system/selinux/package_libselinux_installed/rule.yml]: OK, makes sense. Done.

**Comment by jan-cerny** [comment]
@openscap-ci test this please

**Comment by shawndwells** [comment]
resolves https://github.com/ComplianceAsCode/content/issues/5223

**Pull request merged**